.github/workflows/test.yml

```diff
@@ -1,18 +1,55 @@
+name: Test and build outline-cli
+
 on: [push, pull_request]
-name: Test
+
 jobs:
   test:
+    name: Test
     strategy:
       matrix:
         go-version: [1.19.x]
         os: [ubuntu-latest]
     runs-on: ${{ matrix.os }}
+
     steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
       - name: Install Go
         uses: actions/setup-go@v4
         with:
           go-version: ${{ matrix.go-version }}
+
+      - name: Get dependencies
+        run: go mod download
+
+      - name: Run tests with coverage
+        run: go test -coverprofile=coverage.txt -covermode=atomic ./...
+      
+      - name: Generate coverage report
+        run: go get github.com/axw/gocov/gocov
+        run: gocov convert coverage.txt
```

Comment:
We should also upload it somewhere right? Otherwise it doesn't make sense to generate it...

Reply:
Yes, I wasn't sure if we actually want to check coverage for now, but I'll add some uploading to it. Maybe github has some cool integrated feature where it can read theses coverage files and provide some graph/button. Saw a coverage button in other repos.

Reply:
Guys lets keep CI simple for now IMHO